internal/staticupload/upload.go

```diff
@@ -34,3 +36,15 @@ func (c *Client) Upload(
 	}
 	return output, nil
 }
+
+// Put is a convenience method to save a DEK to AWS S3 Storage by key ID.
```

Comment:
Please keep the convencience methods local to pkg that need them. As far as I can tell this is only `internal/api/repo` so far. 

Also: please don't import `internal/kms` here. `internal/staticupload` should stay independent from our kms backend.

Reply:
our s3 wrapper has the same method and we want to replace all s3 instances with the new client so it will be used. Malte agreed